app/code/Magento/Catalog/Model/Product/Attribute/Source/Layout.php

```diff
@@ -26,14 +26,13 @@ public function __construct(\Magento\Framework\View\Model\PageLayout\Config\Buil
     }
 
     /**
-     * @return array
+     * {@inheritdoc}
```

Comment:
Should be without braces, see https://devdocs.magento.com/guides/v2.0/coding-standards/docblock-standard-general.html#inheritdoc